modules/cnf-image-based-upgrade-configure-container-storage-image-cleanup.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-image-based-upgrade-shared-container-partition
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-configure-container-storage-image-cleanup_{context}"]
+= Configuring the automatic image cleanup of the container storage disk 
+
+Configure when the {lcao} cleans up unpinned images in the `Prep` stage by setting a minimum threshold for available storage space through annotations.
+The default container storage disk usage threshold is 50%.
+
+The {lcao} does not delete images that are pinned in CRI-O or are currently used.
+The Operator selects the images for deletion by starting with dangling images and then sorting the images from oldest to newest that is determined by the image `Created` timestamp.
+
+.Prerequisites
+
+* Create an `ImageBasedUpgrade` CR.
+
+.Procedure
+
+. Increase the threshold to 65% by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-lifecycle-agent annotate ibu upgrade image-cleanup.lca.openshift.io/disk-usage-threshold-percent='65'
+----
+
+.. (Optional) Remove the threshold override by running the following command:
++
+[source,terminal]
+----
+$ oc -n  openshift-lifecycle-agent annotate ibu upgrade image-cleanup.lca.openshift.io/disk-usage-threshold-percent-
+----
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
@amolnar-rh you've repeated a "header type line" that must only occur once in the module. Please delete the line ":_mod-docs-content-type: PROCEDURE" and see if that fixes the problem preventing you from passing the PROW checks.